docs/topics/spider-middleware.rst

```diff
@@ -189,8 +189,9 @@ DepthMiddleware
 .. class:: DepthMiddleware
 
    DepthMiddleware is a scrape middleware used for tracking the depth of each
-   Request inside the site being scraped. It can be used to limit the maximum
-   depth to scrape or things like that.
+   Request inside the site being scraped. It sets `request.meta['depth']`
+   if its not already present in the request and increments it otherwise.
+   It can be used to limit the maximum depth to scrape or things like that.
```

Comment:
I think after the change it sounds like 'It' means `request.meta['depth']`, i.e. request meta['depth'] can be used to limit the maximum depth, not DepthMiddleware. Could you please re-word it, e.g. write DepthMiddleware instead of this 'It'?